src/lib.rs

```diff
@@ -2287,6 +2344,218 @@ impl PlatformAvailability {
     }
 }
 
+// PrettyPrinter _________________________________
+
+/// Pretty prints declarations.
+#[cfg(feature="gte_clang_7_0")]
+#[derive(Debug)]
+pub struct PrettyPrinter<'e> {
+    ptr: CXPrintingPolicy,
+    entity: &'e Entity<'e>,
+}
+#[cfg(feature="gte_clang_7_0")]
+impl<'e> PrettyPrinter<'e> {
+    //- Constructors -----------------------------
+
+    fn from_raw(ptr: CXPrintingPolicy, entity: &'e Entity<'e>) -> Self {
+        assert!(ptr.is_null());
+        PrettyPrinter { ptr, entity }
+    }
+
+    //- Accessors --------------------------------
+
+    /// Pretty print the declaration.
+    pub fn print(&self) -> String {
+        unsafe { utility::to_string(clang_getCursorPrettyPrinted(self.entity.raw, self.ptr)) }
+    }
+
+    //- Properties -------------------------------
```

Comment:
While I like the addition of the `PrettyPrinter` struct to encapsulate pretty printing, I think it would be better to add a `PrintingPolicy` enum to the crate and then replace these individual properties methods with `get_policy` and `set_policy` methods that take the new `PrintingPolicy` enum.

Reply:
The problem with that is there are two different types used. Indentation is a `u8`, and everything else is a `bool`. A `set_property` function is easy enough to do, just have the enum contain the value. A `get-property` is less trivial as it would have to return two different types.

Could do it as a `set_flag` & `get_flag` pair and keep the existing functions for indentation.